src/main/java/io/cdap/e2e/utils/GcpClient.java

```diff
@@ -6,24 +6,25 @@
 import com.google.cloud.bigquery.TableResult;
 import java.io.IOException;
 
+
 public class GcpClient {
+    public static  String query;
```

Comment:
Never use volatile static fields. It's a big anti-pattern. In this case you want `createBigQueryClient()` just to return properly configured `BigQuery` object and `countBqQuery`/`dropBqQuery` can simply use it to run necessary query